src/components/SavedNews/SavedNews.jsx

```diff
@@ -0,0 +1,22 @@
+import NewsCard from "../NewsCard/NewsCard";
+import SavedNewsHeader from "../SavedNewsHeader/SavedNewsHeader";
+import "./SavedNews.css";
+
+function SavedNews({ savedArticles, onRemoveArticle, keywords }) {
+  return (
+    <>
+      <SavedNewsHeader savedArticles={savedArticles} keywords={keywords} />
+      <div className="saved__news-list">
```

Comment:
The `div` with `saved__news-list` should be `ul` according to the semantics because `NewsCard` contains `li` inside the markup.